PowerEditor/src/WinControls/TabBar/TabBar.cpp

```diff
@@ -915,35 +915,29 @@ LRESULT TabBarPlus::runProc(HWND hwnd, UINT Message, WPARAM wParam, LPARAM lPara
 
 		case WM_ERASEBKGND:

```

Comment:
For now use something like this
```c++
case WM_ERASEBKGND:
{
	if (!NppDarkMode::isEnabled())
	{
		break;
	}
	return FALSE;
}

case WM_PAINT:
{
	LONG_PTR dwStyle = GetWindowLongPtr(hwnd, GWL_STYLE);
	if (!NppDarkMode::isEnabled() || !(dwStyle & TCS_OWNERDRAWFIXED))
	{
		break;
	}

	PAINTSTRUCT ps{};
	HDC hdc = _dblBuf.beginPaint(hwnd, &ps);
```

replacing everything until 
`const bool hasMultipleLines = ((dwStyle & TCS_BUTTONS) == TCS_BUTTONS);`